site/public/js/admin-gradeable-updates.js

```diff
@@ -150,7 +150,69 @@ $(document).ready(function () {
                 updateErrorMessage();
             }, updateGradeableErrorCallback);
     });
-});
+    $('#random_peer_graders_list').click(
+        function () {
```

Comment:
It seems to me that there are a whole lot of things going on in this function that don't have anything to do with random grader assignment. Can you speak to what's going on here?

Reply:
Removed Unnecessary Code!